polaris.shopify.com/content/design/index.md

```diff
@@ -1,12 +1,175 @@
+        </Box>
+      </Grid.Cell>
+      <Grid.Cell columnSpan={{xs: 6, sm: 6, md: 6, lg: 8, xl: 8}}>
+        ![Visual: Import, Export, Edit, Print, and other icons that are established](/images/design/design-principles-meaningful-established-symbols@2x.png)
+      </Grid.Cell>
+    </Grid>
+
+  </Card>
+    <Card>
+    <Grid gap="400">
+      <Grid.Cell columnSpan={{xs: 6, sm: 6, md: 6, lg: 4, xl: 4}}>
+        <Box padding="400">
+
+          ## High density
+
+          **Space is optimized while maintaining high usability.**
+
+          Density in the interface adapts to the task: high density is used in data-rich environments, while low density is employed in focused, detailed areas.
+
+        </Box>
+      </Grid.Cell>
+      <Grid.Cell columnSpan={{xs: 6, sm: 6, md: 6, lg: 8, xl: 8}}>
+        ![Visual: Product index page table showing a high density layout](/images/design/design-principles-high-density-task@2x.png)
+      </Grid.Cell>
+      <Grid.Cell columnSpan={{xs: 6, sm: 6, md: 6, lg: 4, xl: 4}}>
+        <Box padding="400">
+
+          Information is structured in a clear visual hierarchy, emphasizing important details through variants like size, weight, and color.
+
+        </Box>
+      </Grid.Cell>
+      <Grid.Cell columnSpan={{xs: 6, sm: 6, md: 6, lg: 8, xl: 8}}>
+        ![Visual: Order detail page example, with a page title, a card with title and badge](/images/design/design-principles-high-density-hierarchy@2x.png)
+      </Grid.Cell>
+      <Grid.Cell columnSpan={{xs: 6, sm: 6, md: 6, lg: 4, xl: 4}}>
+        <Box padding="400">
+
+          Space is used to create groups of related information. Color adds or removes emphasis to make the interface easier to scan and enables efficient decision making.
+
+        </Box>
+      </Grid.Cell>
+      <Grid.Cell columnSpan={{xs: 6, sm: 6, md: 6, lg: 8, xl: 8}}>
+        ![Video: Installed apps card with sub-surfaces for each installed app](/images/design/design-principles-high-density-groups@2x.png)
+      </Grid.Cell>
+    </Grid>
+
+  </Card>
+  <Card>
+    <Grid gap="400">
+      <Grid.Cell columnSpan={{xs: 6, sm: 6, md: 6, lg: 4, xl: 4}}>
+        <Box padding="400">
+
+          ## Tactility
+
+          **Interfaces incorporate a sense of realness.**
+
+          Primary interaction points provide a dramatic visual response to interaction. Variables like position and lighting change to mimic how physical buttons react when pressed. A hover transition is smooth and unobtrusive, whilst feedback from a click feels more dramatic and visceral.
+
+        </Box>
+      </Grid.Cell>
+      <Grid.Cell columnSpan={{xs: 6, sm: 6, md: 6, lg: 8, xl: 8}}>
+        ![Visual: Animation of buttons being hovered and clicked](/images/design/design-principles-tactility-realness@2x.png)
```

Comment:
@tjonx  @zecarlostorre 
[The document](https://docs.google.com/document/d/13LrzMI5SMzDZ3Wm117Yz4olE7PzBCFQh2_B5C07WQlE/edit#heading=h.m850jhu8ex16) specifies the asset as a video, however an image was provided. 

Do we have a requisite animation, or is the doc incorrect?